### DISCUSSION TITLE
0.7.0-stable has been released!

### DISCUSSION THREAD

**Discussion by mihe** [godot-jolt/godot-jolt#573] [Announcements]
This release supports [**Godot 4.1**](https://github.com/godotengine/godot/releases/tag/4.1.1-stable).

## Changelog

These are the notable changes that have been made since [0.6.0-stable](https://github.com/godot-jolt/godot-jolt/releases/tag/v0.6.0-stable) was released. You can also find a list of all the commits [here](https://github.com/godot-jolt/godot-jolt/compare/v0.6.0-stable...v0.7.0-stable).

Any breaking changes are denoted with ⚠️.

### Removed

- ⚠️ Disabled the `JoltDebugGeometry3D` node for all distributed builds. If you still need it, build from source using the `*-development` or `*-debug` configurations.

### Changed

- ⚠️ Ray-casts will no longer hit the back-faces of `ConcavePolygonShape3D` if its `hit_back_faces` parameter is set to `false`, regardless of what the `backface_collision` property of the `ConcavePolygonShape3D` is set to.
- ⚠️ Changed the triangulation of `HeightMapShape3D` to match Godot Physics.

### Fixed

- ⚠️ Fixed regression where a motored `HingeJoint3D` (or `JoltHingeJoint3D`) would rotate counter-clockwise instead of clockwise.
- Fixed issue where ray-casting a `ConcavePolygonShape3D` that had `backface_collision` enabled, you would sometimes end up with a flipped normal.
- Fixed issue where a `CharacterBody3D` using `move_and_slide` could sometimes get stuck when sliding along a wall.
- Fixed issue where attaching a `RigidBody3D` with locked axes to a joint could result in NaN velocities/position and subsequently a lot of random errors being emitted from within Godot.

<hr /><em>This discussion was created from the release <a href='https://github.com/godot-jolt/godot-jolt/releases/tag/v0.7.0-stable'>0.7.0-stable</a>.</em>

**Comment by psrar** [comment]
Amazing work! The only solution that is suitable for my project out of the box for a while. I hope you will continue this work and it will be appreciated by much more people👏👏👏

**Comment by venilark** [comment]
1.0 and the multithreading is getting close and it will be awesome :)
